kgitb/github.py

```diff
@@ -50,27 +50,27 @@ def upsert_comment(url, messages):
                 delete_comment(comment)
             create_comment(url, message)
         if len(comments) == 1:
-            if comments[0]["body"] != message:
+            if comments[0]['body'] != message:
                 update_comment(comments[0], message)
         else:
             create_comment(url, message)
 
 
 def update_comment(comments, body):
-    requests.patch(comments["url"],
-                   params={"access_token": GH_TOKEN},
-                   json={"body": body})
+    requests.patch(comments['url'],
+                   params={'access_token': GH_TOKEN},
+                   json={'body': body})
 
 
 def create_comment(url, body):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)